modules/standard/VectorizingIterator.chpl

```diff
@@ -39,7 +39,7 @@
  */
 pragma "error mode fatal"
 pragma "unsafe"
-pragma "module included by default"
+deprecated "The 'VectorizingIterator' module has been deprecated; please use 'foreach' loops instead"
```

Comment:
Nit, but I might say "please use a 'foreach' loop instead" instead of the plural "loops", but I don't feel strong enough about it to put up any sort of fight.

Reply:
If this were the deprecation warning on the iterator itself, I could get behind this, but since it's an error for the module as a whole where the user may be planning to apply `vectorizeOnly()` to multiple loops, I think this form makes sense.  (Like "don't use this module's contents, use this language concept instead").  But let me know if you're seeing something that I'm not.